### PR TITLE
News section documentation

### PR DESCRIPTION
**Pull request by ruslanmalogulko**
add a news section django cms 3 2 0 rc8 documentation 2015-11-27 17-29-53

### CODE REVIEW

File: docs/user/tutorial/50-news.rst [hunk 1]
```diff
@@ -8,6 +8,40 @@ Blog* application.
...
+Just as we did with Aldryn People, we should create an apphook to create a landing page for the
+news items.
+
+We'll create a *News* page, and add an apphook for the Aldryn News & Blog application to it.
+
+* create and publish a new page called News
+* open page settings
+
+    .. image:: /user/tutorial/images/page-settings-button.png
+        :alt: 'page settings button'
```
Comment (evildmp) [inline docs/user/tutorial/50-news.rst]: Don't forget to change alt text as required. It doesn't need to be perfect of course, I can always tweak it before merging.

File: docs/user/tutorial/50-news.rst [hunk 2]
```diff
@@ -66,40 +101,20 @@ Do the same for a second item:
...
 * switch to Edit mode
 * enter Structure mode
 * select **Add plugin** on *Content* placeholder
 * select *Latest articles* plugin from list
-* select the existing Apphook (hopefully we can get rid of this step)
+* select the existing Apphook
 * **Save**
 * see news articles inserted into home page
+
+    .. image:: /user/tutorial/images/home-page-news-articles.png
```
Comment (evildmp) [inline docs/user/tutorial/50-news.rst]: For this image, can we show the whole home page with the plugin in its context please?

**Comment by evildmp** [comment]
@ruslanmalogulko comments made thanks.

File: docs/user/tutorial/50-news.rst (hunk 2 above)
Comment (darioalbanesi) [inline docs/user/tutorial/50-news.rst]: If so, I should probably do the same for the people page. Currently I'm only showing the people but not the whole page. @evildmp what do you think?

File: docs/user/tutorial/50-news.rst (hunk 1 above)
Comment (ruslanmalogulko) [inline docs/user/tutorial/50-news.rst]: please provide me with correct alt text here

**Pull request merged**